helios-testing/src/main/java/com/spotify/helios/testing/TemporaryJobs.java

```diff
@@ -87,7 +87,7 @@
   private static final Prober DEFAULT_PROBER = new DefaultProber();
   private static final String DEFAULT_LOCAL_HOST_FILTER = ".+";
   private static final String DEFAULT_PREFIX_DIRECTORY = "/tmp/helios-temp-jobs";
-  private static final String DEFAULT_TEST_REPORT_DIRECTORY = "./helios-reports/test";
+  private static final String DEFAULT_TEST_REPORT_DIRECTORY = "target/helios-reports/test";
```

Comment:
This assumes you're running with maven or any build system that relies on the maven directory structure